File: gc-commons/gc-common/src/main/java/com/gc/common/utils/UUIDGenerator.java
package com.gc.common.utils;

import java.util.UUID;

/**
 * UUID工具类
 * @author shizhongming
 * 2020/1/8 7:54 下午
 */
public class UUIDGenerator {

    /**
     * 生成UUID
     * @author shizhongming
     * @return
     */
    public static String getUUID() {
        var s = UUID.randomUUID().toString();
        s = s.replace("-", "");
        return s;
    }

    /**
     *
     * @param a
     */
    public void abc(String a) {

    }
}

File: smart-common/src/main/kotlin/com/smart/common/utils/UUIDGenerator.kt
package com.smart.common.utils

import java.util.*

/**
 * UUID 工具类
 * @author ming
 * 2019/6/12 上午10:00
 */
object UUIDGenerator {

    /**
     * 生成UUID
     */
    @JvmStatic fun getUUID(): String {
        var s = UUID.randomUUID().toString()
        s = s.replace("-", "")
        return s
    }

    /**
     * 生成UUID 指定位数
     * @param number 位数
     */
    @JvmStatic fun getUUID(number: Int): Array<String?>? {
        if (number < 1) {
            return null
        }
        val ss = arrayOfNulls<String>(number)
        for (i in 0 until number) {
            ss[i] = getUUID()
        }
        return ss
    }
}